### PR TITLE
Sound: Fix Faux Sound incorrect values

### PR DESCRIPTION
**Pull request by Kamin4ri**
* Faux123's Sound Control driver uses a separate range of integers for negative values (226 to 255, -30 to -1 respectively), since unsigned ints are being used (unsigned ints can't be less than 0). Also, Headphone Poweramp Gain's value is in the 32-44 range (in a descending order from 6 to -6, with zero being 38). This caused the app to write and read incorrect values to/from the sysfs control files. This commit aims to fix that.

* I also fixed a typo ("micropone") in a variable.

### CODE REVIEW

File: app/src/main/java/com/grarak/kerneladiutor/utils/kernel/sound/Sound.java [hunk 1]
```diff
@@ -178,28 +205,59 @@ public static boolean hasCamMicrophoneGain() {
...
+            value = String.valueOf(newGain + 256);
+            fauxRun(value + " " + value, HEADPHONE_GAIN, HEADPHONE_GAIN, context);
+        }
     }
 
     public static String getHeadphoneGain() {
         String value = Utils.readFile(HEADPHONE_GAIN);
-        return value.contains(" ") ? value.split(" ")[0] : value;
+        String strGain = null;
+        if (value.contains(" "))
```
Comment (Grarak) [inline app/src/main/java/com/grarak/kerneladiutor/utils/kernel/sound/Sound.java]: Add braces here

File: app/src/main/java/com/grarak/kerneladiutor/utils/kernel/sound/Sound.java [hunk 2]
```diff
@@ -178,28 +205,59 @@ public static boolean hasCamMicrophoneGain() {
...
+            strGain = value.split(" ")[0];
```
Comment (Grarak) [inline app/src/main/java/com/grarak/kerneladiutor/utils/kernel/sound/Sound.java]: Same

File: app/src/main/java/com/grarak/kerneladiutor/utils/kernel/sound/Sound.java [hunk 3]
```diff
@@ -178,28 +205,59 @@ public static boolean hasCamMicrophoneGain() {
...
+        else
+            strGain = value;
```
Comment (Grarak) [inline app/src/main/java/com/grarak/kerneladiutor/utils/kernel/sound/Sound.java]: Same

**Comment by Kamin4ri** [comment]
Do you mean this?
```java
String strGain = (value.contains(" ") ? value.split(" ")[0] : value);
```

**Comment by Kamin4ri** [comment]
@Grarak: I added a previously FauxSound-exclusive feature which is the other Headphone Amp Gain (TPA6165 extension). We could also have individual controls for each channel, and the "telephony extensions", to match FauxSound's feature set, but thats up to you. I changed quite a load of stuff here already.

**Comment by Kamin4ri** [comment]
Sorry for the absurd amount of commits. I'm tired so i'm doing a lot of derps here -.-

**Comment by Grarak** [comment]
squash them into one

**Pull request closed without merging**